architecture/additional_concepts/admission_controllers.adoc

```diff
@@ -32,3 +32,4 @@ ifdef::openshift-dedicated[]
 - link:../../admin_guide/osd_build_defaults_overrides.html[Configuring Global Build Defaults and Overrides]
 endif::[]
 - link:../../admin_guide/scheduler.html#controlling-pod-placement[Controlling Pod Placement]
+- link:../../admin_guide/overcommit.html[Configuring Overcommit]
```

Comment:
@sosiouxme Looks like we don't include the Overcommitting topic in the Dedicated version of the Cluster Admin guide, so this new line would need to get ifdef'd appropriately (or move it up into the other dedicated ifdef). But to your point in the other comment, if you don't think it warrants a highlight here, fine to omit completely.